public/app/features/dashboard/components/SaveDashboard/useDashboardSave.tsx

```diff
@@ -49,6 +49,7 @@ export const useDashboardSave = (dashboard: DashboardModel) => {
       reportInteraction(`grafana_dashboard_${dashboard.id ? 'saved' : 'created'}`, {
```

Comment:
Currently, when duplicating a dashboard, we get a `grafana_dashboard_saved` event instead of a created one (although it is a new dashboard we are creating).

I suppose this is because this `dashboard.id` check will return the original dashboard that was duplicated? That also explains why the `name` prop for the event is always the original dashboard not the new one (cf screenshot).

An easy fix would be to report a different interaction entirely (`grafana_dashboard_copied` ) in the case of copy. WDYT? 

![Screenshot 2023-01-31 at 13 55 14](https://user-images.githubusercontent.com/42030685/215766004-0ceef9cc-b833-408b-bf94-0284958950b4.png)

Reply:
sure 👍